contracts/upgradeable_contracts/arbitrary_message/VersionableAMB.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity 0.4.24;
+
+import "../VersionableBridge.sol";
+
+contract VersionableAMB is VersionableBridge {
+    // bridge version as a single 4-bytes number, included into every outgoing relay request
+    uint32 internal constant ENCODED_BRIDGE_VERSION = 0x00040000;
```

Comment:
Seems that it is more logical to move this constant to `ArbitraryMessage.sol` since it reflects the version of how the message is packed. That's why consider to rename it to `MESSAGE_PACKING_VERSION` or something like this.

Reply:
I think it makes sense to keep all versions-related stuff in one place, it will also help us to not forget to change this constant when bridge version is changed.

Another point is related to libraries in solidity, seems that it is not possible to access constants from library. I think that introducing a public getter is a big overhead in our case, so I propose to leave it in `VersionableAMB.sol`.

Reply:
Got you. Please add in the library that if the format of the message is changed, it is necessary to update the message packing version as well.

Reply:
Ok, added a note